src/index.js

```diff
@@ -0,0 +1,243 @@
+const inquirer = require("inquirer");
+const {
+  viewDepartments,
+  viewRoles,
+  viewEmployees,
+  addDepartment,
+  addRole,
+  addEmployee,
+  updateEmployee,
+  deleteRole,
+  deleteDepartment,
+  deleteEmployee,
+  viewBudget,
+  viewByDepartment,
+  viewByManager,
+} = require("./db/queries");
+const { initialQuestion, deptQuestion } = require("./questions");
+const Db = require("./middleware/db");
+const {
+  generateDepartmentChoices,
+  generateRoleChoices,
+  generateManagerChoices,
+  generateEmployeeChoices,
+} = require("./utils");
+
+const title = `
+                                                                    
+▄███▄   █▀▄▀█ █ ▄▄  █    ████▄ ▀▄    ▄ █▀▄▀█ ▄███▄      ▄     ▄▄▄▄▀ 
+█▀   ▀  █ █ █ █   █ █    █   █   █  █  █ █ █ █▀   ▀      █ ▀▀▀ █    
+██▄▄    █ ▄ █ █▀▀▀  █    █   █    ▀█   █ ▄ █ ██▄▄    ██   █    █    
+█▄   ▄▀ █   █ █     ███▄ ▀████    █    █   █ █▄   ▄▀ █ █  █   █     
+▀███▀      █   █        ▀       ▄▀        █  ▀███▀   █  █ █  ▀      
+          ▀     ▀                        ▀           █   ██         
+                                                                    
+               ▄▄▄▄▀ █▄▄▄▄ ██   ▄█▄    █  █▀ ▄███▄   █▄▄▄▄          
+            ▀▀▀ █    █  ▄▀ █ █  █▀ ▀▄  █▄█   █▀   ▀  █  ▄▀          
+                █    █▀▀▌  █▄▄█ █   ▀  █▀▄   ██▄▄    █▀▀▌           
+               █     █  █  █  █ █▄  ▄▀ █  █  █▄   ▄▀ █  █           
+              ▀        █      █ ▀███▀    █   ▀███▀     █            
+                      ▀      █          ▀             ▀        `;
+
+const db = new Db({
```

Comment:
Consider moving the db options into the Db class and just passing in the database name. This will make it more reusable.

Reply:
will do thank you